docs/topics_5/installation.adoc

```diff
@@ -78,7 +78,9 @@ ifeval::["{context}" != "web-console-openshift"]
 +
 [NOTE]
 ====
-If you are installing on a Windows operating system, extract the `.zip` file to a folder named `mta` to avoid a `Path too long` error.
+If you are installing on a Windows operating system:
+. Extract the `.zip` file to a folder named `mta` to avoid a `Path too long` error.
+. During extraction, you might receive a message in the *Confirm file replace* window that asks if you would like to replace a duplicated file. If this occurs, select *Yes*. Replacing the file has no effect on the usability of the application.
 ====
```

Comment:
Select "Yes"? That's more likely to be a button than a menu item or list item. Buttons are clicked, not selected.

I think it's a bit too long. Suggested rewrite:

`. If a *Confirm file replace* window is displayed, click *Yes to all*.`

Clicking 'Yes to all' means they don't have to click over and over. You don't need to tell them that replacing the file does not affect usability because that is assumed. Personally, I prefer "If a window appears", but IBM style guide says we can't use the word "appears" because it sounds magical. So we have to say "is displayed" or "opens".

Reply:
I added "...during extraction" to the if clause. Let me know what you think.

Reply:
I don't like it. I think "During extraction" is unnecessary.